CHANGELOG.md

```diff
@@ -82,6 +82,11 @@ Nothing yet.
 
 ### Added
 
+- Add new feature gate `RewriteURIs` to enable/disable the `konghq.com/rewrite`
+  annotation (default disabled).
+  [#4360](https://github.com/Kong/kubernetes-ingress-controller/pull/4360)
+- `konghq.com/rewrite` annotation has been introduced to manage URI rewriting.
+  [#4360](https://github.com/Kong/kubernetes-ingress-controller/pull/4360)
```

Comment:
Please create a new `## Unreleased` section and add your CHANGELOG entries there.